flake.nix

```diff
@@ -2,7 +2,7 @@
   description = "Kontrol";
 
   inputs = {
-    kevm.url = "github:runtimeverification/evm-semantics/v1.0.307";
+    kevm.url = "github:runtimeverification/evm-semantics/noah/contract-initialization-2";
     nixpkgs.follows = "kevm/nixpkgs";
     k-framework.follows = "kevm/k-framework";
```

Comment:
This change should be backed out too.

Reply:
It also depends on this branch, https://github.com/runtimeverification/evm-semantics/pull/2096, although I will back out these changes here now because the checks have passed.